### DISCUSSION TITLE
EventLoopWindowTarget::exit() doesn't immediately stop the event loop

### DISCUSSION THREAD

**Discussion by ItsSunnyMonster** [rust-windowing/winit#3322] [Q&A]
I didn't file this as an issue because I don't know if I missed something or if this is the intended behaviour. When I call `EventLoopWindowTarget::exit()` in my event loop after all windows have been closed, on Windows, I would receive another bunch of events after that, including another `AboutToWait` event where I draw a frame. On macOS a `NewEvents(WaitCancelled)` event still gets emitted but that doesn't affect anything because I don't handle that event yet. Why is this?

**Comment by daxpedda** [comment]
Moved to #3323.
